modules/page-memory/src/integrationTest/java/org/apache/ignite/internal/pagememory/tree/ItBplusTreePageMemoryTest.java

```diff
@@ -105,13 +99,11 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInfo;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 /**
- * Class to test the {@link BplusTree}.
+ * An abstract class for testing {@link BplusTree} using different implementations of {@link PageMemory}.
  */
-@ExtendWith(ConfigurationExtension.class)
-public class ItBplusTreeSelfTest extends BaseIgniteAbstractTest {
+public abstract class ItBplusTreePageMemoryTest extends BaseIgniteAbstractTest {
```

Comment:
I guess since this class is now abstract and does not contain any tests, it should not start with `It`

Reply:
Fix it